models/diffusion_networks.py

```diff
@@ -93,6 +93,21 @@ def define_G(
             use_scale_shift_norm=True,
         )
 
+    elif G_netG == "dit":
+        denoise_fn = DiT(
+            input_size=data_crop_size,
+            patch_size=8,
+            in_channels=model_input_nc * 2,
+            out_channels=model_output_nc,
+            hidden_size=320,
+            depth=28,
+            num_heads=16,
+            mlp_ratio=4.0,
+            class_dropout_prob=0.1,
+            num_classes=0,  # 1000,
```

Comment:
This needs to be set to 1 by default, and the number of conditioning classes otherwise.

Reply:
For now, this option is uselesse because we don't use conditionning during noise prediction.